common/workunit/workunit.cpp

```diff
@@ -2956,7 +3112,7 @@ void WuScopeFilter::finishedFilter()
     if (include.nestedDepth == 0)
     {
         if (scopeFilter.compareDepth(3) > 0)    // 3 is larger than any scope in the filter
-            sourceFlags &= ~(SSFsearchGraphStats);
+            sourceFlags &= ~(SSFsearchGraphStats|SSFsearchExceptions);
```

Comment:
Why is this true?  I would expect to always need to search the exceptions.

Reply:
Originally, I thought all issues would be related to activities.  This is not true, so this change is needed.